src/components/IKUpload/index.tsx

```diff
@@ -21,7 +40,7 @@ export default class IKUpload extends ImageKitComponent {
       customCoordinates,
       responseFields,
       onError,
-      onSuccess
+      onSuccess,
```

Comment:
Type definitions of props are not accurate.
<img width="577" alt="image" src="https://user-images.githubusercontent.com/45416977/190081709-938dc14b-1e81-427d-be80-bd0e104caa4f.png">